README.adoc

```diff
@@ -86,7 +86,7 @@ endif::[]
 
 These are the minimal steps to convert your AsciiDoc documents to HTML with the asciidoctor-maven-plugin.
 
-. Place your Asciidoc sources in [.path]_/src/docs/asciidoc_.
+. Place your Asciidoc sources in [.path]_src/docs/asciidoc_.
```

Comment:
Good catch! While you are on this line, could you also change Asciidoc to AsciiDoc?

Reply:
Sure - https://github.com/asciidoctor/asciidoctor-maven-plugin/pull/592/commits/96af436ebe20f017f9d3b0ee6361d00c964c4ed1